modules/images.py

```diff
@@ -311,7 +311,7 @@ def apply_filename_pattern(x, p, seed, prompt):
         x = x.replace("[cfg]", str(p.cfg_scale))
         x = x.replace("[width]", str(p.width))
         x = x.replace("[height]", str(p.height))
-        x = x.replace("[styles]", sanitize_filename_part(", ".join([x for x in p.styles if not x == "None"]), replace_spaces=False))
+        x = x.replace("[styles]", sanitize_filename_part(", ".join([x for x in p.styles if not x == "None"]) or "No styles", replace_spaces=False))

```

Comment:
I agree with the early comment that having [styles] return "None, None" if styles aren't in use, wasn't very elegant, so I instead made it return nothing. But then that exposed a problem when a commenter used `[prompt_no_styles] + [styles]` as a directory naming pattern. Running strip() on this line wouldn't have prevented the trailing space because it was built into the user's input. Returning "No styles" makes the issue moot (unless they had a trailing space in the style name, I guess?), and I think it is an overall more helpful return value if a user is using [styles] in their naming patterns.